docker/0.90-2/final/Dockerfile.cpu

```diff
@@ -1,7 +1,56 @@
 FROM xgboost-container-base:0.90-2-cpu-py3
+ENV SAGEMAKER_XGBOOST_VERSION 0.90-2
 
-LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+########################
+# Install dependencies #
+########################
+COPY requirements.txt /requirements.txt
+RUN pip install -r /requirements.txt && rm /requirements.txt
+
+###########################
+# Copy wheel to container #
+###########################
+COPY dist/sagemaker_xgboost_container-2.0-py2.py3-none-any.whl /sagemaker_xgboost_container-1.0-py2.py3-none-any.whl
+RUN pip install --no-cache /sagemaker_xgboost_container-1.0-py2.py3-none-any.whl && \
+    rm /sagemaker_xgboost_container-1.0-py2.py3-none-any.whl
+
+##############
+# DMLC PATCH #
+##############
+# TODO: remove after making contributions back to xgboost for tracker.py
+COPY src/sagemaker_xgboost_container/dmlc_patch/tracker.py \
+   /miniconda3/lib/python3.6/site-packages/xgboost/dmlc-core/tracker/dmlc_tracker/tracker.py
+
+# Include DMLC python code in PYTHONPATH to use RabitTracker
+ENV PYTHONPATH=$PYTHONPATH:/miniconda3/lib/python3.6/site-packages/xgboost/dmlc-core/tracker
+
+#######
+# MMS #
+#######
+# Create MMS user directory
+RUN useradd -m model-server
+RUN mkdir -p /home/model-server/tmp
+RUN chown -R model-server /home/model-server
```

Comment:
Do these all need to be separate `RUN` commands (as opposed to chaining)? As each `RUN` instruction will execute any commands in a new layer on top of the current image and commit the results. Just trying to understand if we are doing this on purpose.

Reply:
No reason not to chain the mkdir and chown, made the change.